### PR TITLE
Changed param names for azrbac

### PR DESCRIPTION
**Pull request by Anumita**
---

Changing the param names for users in azrbac for more readability

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by Anumita** [comment]
@fengzhou-msft , could you raise the release PR for connectedk8s after this PR is merged?

File: src/connectedk8s/azext_connectedk8s/_help.py [hunk 1]
```diff
@@ -103,7 +103,7 @@
   - name: Enables the Cluster-Connect feature.
     text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features cluster-connect
   - name: Enable Azure RBAC feature.
-    text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features azure-rbac --client-id clientID --client-secret="clientSecret" --skip-azure-rbac-list "user1@domain.com,spn_oid"
+    text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features azure-rbac --app-id appID --app-secret="appSecret" --skip-azure-rbac-list "user1@domain.com,spn_oid"
```
Comment (shashankbarsin) [inline src/connectedk8s/azext_connectedk8s/_help.py:106]:
```suggestion
    text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features azure-rbac --app-id appId --app-secret="appSecret" --skip-azure-rbac-list "user1@domain.com,spn_oid"
```

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def load_arguments(self, _):
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
         c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to enable.')
-        c.argument('azrbac_client_id', options_list=['--client-id'], arg_group='Azure RBAC', help='Client ID for enabling Azure RBAC. Specify when enabling azure-rbac.')
```
Comment (raunakpandya) [inline src/connectedk8s/azext_connectedk8s/_params.py:59]: >client [](start = 56, length = 6)

We need to fix the references for client in custom.py as well.. Like

telemetry.set_exception(exception='Client ID or secret is not provided for Azure RBAC', fault_type=consts.Client_Details_Not_Provided_For_Azure_RBAC_Fault,
                                    summary='Client id, client secret is required to enable/update Azure RBAC feature')

Comment (Anumita) [reply src/connectedk8s/azext_connectedk8s/_params.py:59]: done

**Review by raunakpandya — APPROVED** [review]
:shipit:

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**